src/content/en/tools/workbox/guides/migrate.md

```diff
@@ -0,0 +1,224 @@
+project_path: /web/tools/workbox/_project.yaml
+book_path: /web/tools/workbox/_book.yaml
+description: A guide to migrating from sw-precache or sw-toolbox to Workbox.
+
+{# wf_updated_on: 2018-01-25 #}
+{# wf_published_on: 2018-01-25 #}
+{# wf_blink_components: N/A #}
+
+# Migrate from sw-precache or sw-toolbox {: .page-title }
+
+{% include "web/tools/workbox/_shared/alpha.html" %}
+
+Developers who have previously used [`sw-precache`](https://github.com/GoogleChromeLabs/sw-precache)
+and/or [`sw-toolbox`](https://github.com/GoogleChromeLabs/sw-toolbox) have a straightforward upgrade
+path to the Workbox family of libraries. Upgrading to Workbox will provide a modern, extensible
+service worker experience with improved debugging and developer ergonomics.
+
+## Modifications to your existing configuration
+
+Regardless of which specific build tools you're migrating from, keep in mind the following name
+changes and deprecations when putting together your equivalent Workbox configuration.
+
+### Renamed options
+
+- The `runtimeCaching` config parameter has been
+[updated](https://github.com/GoogleChrome/workbox/issues/1096#issue-278581349) to take a more
```

Comment:
this link is really out of context for the what you are describing here.

Reply:
+1

(Filling in for Pete today, by the way.)